src/pages/General.vue

```diff
@@ -50,4 +58,13 @@ export default {
 
 <!-- Add "scoped" attribute to limit CSS to this component only -->
 <style scoped>
+
+.general li {
+  margin-bottom: .5em;
+}
+
+.general p {
```

Comment:
We should get rid of the `p` styles in `_typography.css`; as far as I can tell that's not even copied from dashboard.

Reply:
brought over an update to this file that no longer has the font size change.